src/components/Card/Card.module.scss

```
@@ -33,14 +41,21 @@
     padding: $space-xl $space-xs;
     background-color: $white-color;
     flex: 1;
+    html[data-theme="dark"] & {
+      background-color: $text-secondary-light;
+    }
```

в таких случаях, возможно, проще будет использовать миксин, но уже можно не менять
```
@mixin dark {
  html[data-theme="dark"] & {
    @content;
  }
}
```
```suggestion
    @include dark {
      background-color: $text-secondary-light;
    }
```

---

да, я уже понял, что если что-то переиспользуется то лучше это выносить компоненты, обертки или миксины )